src/app/detail/detail.component.html

```diff
@@ -0,0 +1,13 @@
+
+<div class="detail-wrapper">
+  <h3>Detail</h3>
+
+  <br>
+
+  <div *ngFor="let colDef of this.gridUtil.COLUMN_DEFINITIONS">
+    <div *ngIf="colDef.field">
+      <p><b>{{colDef.field}}</b> &nbsp;&nbsp;:&nbsp;&nbsp;{{ this.detailRecord[colDef.field] }}</p>
```

Comment:
Can you please use some nice Material Design widgets? Start with a simple [table](https://material.angular.io/components/table/overview).